### PR TITLE
Add error page for custom 404 error messages

### PR DESCRIPTION
**Pull request by joelfranusic-okta**
This will allow us to have a "branded" error page.

### CODE REVIEW

**Comment by joelfranusic-okta** [comment]
@mystiberry-okta @johngronberg-okta for your review.

File: _source/error.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+---
+<section>
+<h3>404 Page Not Found</h3>
+
+<p>The web page or file you were trying to reach cannot be found. It may have been moved to a different location or removed because the information it contained is no longer valid.</p>
+
+<p>Common errors that can result in broken links include:</p>
+
+<ul>
+  <li>Truncated URL's.<br/>Some email applications may break a URL due to line breaks. You may need to cut and paste a URL from your email application into your browser's location field.</li>
```
Comment (johngronberg-okta) [inline _source/error.html]: Is URL's or URLs proper?

Comment (mystiberry-okta) [inline _source/error.html]: URLs plural is most correct (though you see URL's frequently).

**Comment by joelfranusic-okta** [comment]
Fixed plural, any other things to fix @mystiberry-okta or @johngronberg-okta?

**Comment by johngronberg-okta** [comment]
LGTM

**Comment by mystiberry-okta** [comment]
Shipit!  You *could* remove "as printed" (who knows where they got the URL), but not required.

**Pull request merged**